src/Traits/OrderItemCustomerOptionCapableTrait.php

```diff
@@ -74,7 +73,7 @@ public function getCustomerOptionConfigurationAsSimpleArray(): array
     public function setUnitPrice(int $unitPrice): void
     {
         $customerOptionAwareUnitPrice = $this->applyConfigurationPrices($unitPrice);
-        $this->unitPrice = $customerOptionAwareUnitPrice;
+        $this->unitPrice              = $customerOptionAwareUnitPrice;
```

Comment:
This needs to be integrated with the SpecialPricingPlugin proper. This comment is just informational.

Reply:
This is not part of the pull request.

Reply:
:+1: